cms/templates/cms/home_page.html

```diff
@@ -68,180 +74,169 @@
             <div class="sub-banner-text">
-              <h3>What is the MicroMasters<span class="trademark">&reg;</span> Program Credential?</h3>
-            <div class="youtubevideowrap">
-              <div class="video-container">
-                <iframe width="560" height="315" src="https://www.youtube.com/embed/7lVvOSJpFGY" frameborder="0" allowfullscreen></iframe>
-              </div>
-            </div>
-              <p>A MicroMasters program credential is a professional and academic credential for online learners from
-                anywhere in the world. To earn the credential, learners must pass an integrated set of MIT graduate-level courses on edX.org,
-                and one or more proctored exams. This credential is valuable in and of itself.  Additionally,
-                credential-holders can then apply for an accelerated, on campus, master’s degree program at MIT
-                or other universities.
-                {% if benefits_url %}
-                  All MicroMasters program credential holders are also considered
-                  <a href="{{benefits_url}}">affiliates of the MIT Alumni Association</a>.
-                {% endif %}
+              <h2>MITx MicroMasters<span class="trademark">&reg;</span> Programs</h2>
+            
+              <p>Advance your career or accelerate your Master’s degree with a graduate-level digital credential from MIT.</p>
+              <p>The MicroMasters program credential from MIT Open Learning is a <b>professional and academic credential</b> for online learners
+                 from anywhere in the world who seek focused, accelerated advancement.</p> 
+              <p><b>Enroll</b> in a program—no admission required—and take 
+                 a series of graduate-level online courses, taught by MIT instructors, through edX.</p> 
+              <p><b>Earn</b> a program credential by 
+                 completing the course and passing one or more proctored exams.</p>
+              <p>   
+                 <b>Enjoy</b> the credential benefits: Credential earners can also apply for 
+                 an accelerated master’s degree program at MIT and other <a href="/pathways-graduate-programs/">pathway schools</a>; and include your credential on professional 
+                 profiles. MicroMasters program credential earners also become <a href="{{benefits_url}}">affiliates of the MIT Alumni Association</a>.
               </p>
-              <p>MicroMasters program courses offer the same learning and challenge as MIT courses.</p>
-              <div class="sub-banner-logos">
-                <div class="sub-banner-logos-text">All MITx MicroMasters program courses are delivered on</div>
-                <img src="{% static 'images/edx_logo.svg' %}" alt="edX" class="edx-logo">
-                <img src="{% static 'images/micromasters_logo.png' %}" alt="MITx" class="micromasters-logo">
+              <ul class="current-programs-list">
```

Comment:
Top and bottom margins differ, kindly align them with invisio.

Reply:
I think this is worth doing, if it's not too much trouble.

Reply:
@HamzaIbnFarooq a lot of invision design things are not aligned to the requirements as suggested by @pdpinch.

Reply:
Sorry for the inconvenience @umarmughal824, I didn't know about those earlier. Thanks for the great work 👍

Reply:
it's okay. thanks for your review. :)

Reply:
@HamzaIbnFarooq thanks for pointing it out. After comparing both, I also found that it's worthy to do.